ui/pages/settings/experimental-tab/experimental-tab.test.js

```diff
@@ -79,4 +79,24 @@ describe('ExperimentalTab', () => {
       expect(setPetnamesEnabled).toHaveBeenCalledWith(false);
     });
   });
+
+  it('should enable redesigned confirmations', async () => {
```

Comment:
I wonder if we should add an e2e test that opens a bunch of signatures from the test dApp and checks which version is displayed based on the toggle state

Reply:
We can add that validation on existing e2e tests no?
What we could do as well is adding an e2e test with the toggle and env var on for all different confirmations. And then as soon as we migrate a new confirmation type, we would need to update that test.

Reply:
Exactly, that's what I meant with my suggestion.

Reply:
This is sort of temporary feature and will go away as we move to new designs, this I am bit reluctant to add e2e for it.